### DISCUSSION TITLE
how to move the Back to top button to the bottom right corner

### DISCUSSION THREAD

**Discussion by js2ymmks** [squidfunk/mkdocs-material#4215] [Q&A] [answered]
Hello.
Could you please tell me a good way to move the Back to top button to the bottom right corner? I have tried several ways but could not achieve it and am having trouble getting it to work.

**Comment by squidfunk** [comment, marked as answer]
You can use [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css) to change the appearance and position of the back-to-top button.

**Reply by js2ymmks** [reply]
Thank you for your answers.
Overwritten with a few modifications to the CSS.

```
  [dir="ltr"] .md-top {
    margin-left: 75vw;
    margin-top: calc(80vh - 64px);
  }
```

```Back to top``` text is displayed incorrectly when the width is narrowed, so ```ja.html``` has been rewritten.
